push/templates/push/validate.html

```diff
@@ -0,0 +1,19 @@
+{% extends "dashboard/base.html" %}
+
+{% block content %}
+    {% if not request.user.is_authenticated %}
+        {% include 'dashboard/includes/authenticate.html' %}
```

Comment:
Yeah, I wanted to use that approach instead. I was having trouble implementing it, but I'll give it another look.